src/app/[lng]/gas/page.tsx

```diff
@@ -1,7 +1,25 @@
 import App from '@/app/ui/app/App';
+import { WidgetContainer, Widgets } from '@/components/Widgets';
+import { Widget } from '@/components/Widgets/Widget';
+import { getCookies } from '@/app/lib/getCookies';
 
-const Page = async () => {
-  return <App starterVariant="refuel" />;
+const Page = () => {
+  const variant = 'refuel';
```

Comment:
We have this TabsMap.Refuel.variant. Wdyt?

Reply:
https://github.com/lifinance/jumper.exchange/pull/1023#discussion_r1639450829